pg/src/main/java/org/jdbi/v3/pg/PostgresJdbiPlugin.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.pg;
+
+import org.jdbi.v3.Handle;
+import org.jdbi.v3.spi.JdbiPlugin;
+
+public class PostgresJdbiPlugin implements JdbiPlugin {
+    @Override
+    public Handle customizeHandle(Handle handle) {
+        handle.registerArgumentFactory(new SqlArrayArgumentFactory());
+        handle.registerColumnMapper(new SqlArrayMapperFactory());
```

Comment:
Should we get rid of `customizeHandle` to remove ambiguity? I could see that becoming a papercut for users.